rest-api-samples/postman/association-api/WSO2 Identity Server - User Account Associations API Definition.postman_collection.json

```diff
@@ -0,0 +1,610 @@
+						"url": {
+							"raw": "https://localhost:9443/t/carbon.super/api/users/v1/2e053351-0d69-476e-81df-04a9cfdfb50e/federated-associations",
+							"protocol": "https",
+							"host": [
+								"localhost"
+							],
+							"port": "9443",
+							"path": [
+								"t",
+								"carbon.super",
+								"api",
+								"users",
+								"v1",
+								"2e053351-0d69-476e-81df-04a9cfdfb50e",
+								"federated-associations"
+							]
+						},
+						"description": "This API is used to retrieve the federated associations of the user.\n\n  <b>Permission required:</b>\n* /permission/admin/manage/identity/user/association/view\n"
+					},
+					"response": []
+				},
+				{
+					"name": "Delete user's all user federated associations",
+					"request": {
+						"auth": {
+							"type": "basic",
+							"basic": [
+								{
+									"key": "password",
+									"value": "admin",
+									"type": "string"
+								},
+								{
+									"key": "username",
+									"value": "admin",
+									"type": "string"
+								}
+							]
+						},
+						"method": "DELETE",
+						"header": [
+							{
+								"key": "Accept",
+								"value": "application/json"
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json"
+							}
+						],
+						"body": {},
+						"url": {
+							"raw": "https://localhost:9443/t/carbon.super/api/users/v1/UFJJTUFSWS9qb2hu/federated-associations",
+							"protocol": "https",
+							"host": [
+								"localhost"
+							],
+							"port": "9443",
+							"path": [
+								"t",
+								"carbon.super",
+								"api",
+								"users",
+								"v1",
+								"UFJJTUFSWS9qb2hu",
+								"federated-associations"
+							]
+						},
+						"description": "This API is used to delete all federated associations of the  user.\n\n<b>Permission required:</b>\n  * /permission/admin/manage/identity/user/association/delete\n"
+					},
+					"response": []
+				},
+				{
+					"name": "Delete user's federated association",
+					"request": {
+						"method": "DELETE",
+						"header": [
+							{
+								"key": "Accept",
+								"value": "application/json"
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json"
+							}
+						],
+						"body": {},
+						"url": {
+							"raw": "https://localhost:9443/t/carbon.super/api/users/v1/UFJJTUFSWS9qb2hu/federated-associations/UFJJTUFSWS9qb2hu",
+							"protocol": "https",
+							"host": [
+								"localhost"
+							],
+							"port": "9443",
+							"path": [
+								"t",
+								"carbon.super",
+								"api",
+								"users",
+								"v1",
+								"UFJJTUFSWS9qb2hu",
+								"federated-associations",
+								"UFJJTUFSWS9qb2hu"
+							]
+						},
+						"description": "This API is used to delete a federated association of the user.\n\n<b>Permission required:</b>\n  * /permission/admin/manage/identity/user/association/delete\n"
+					},
+					"response": []
+				}
+			]
+		}
+	]
+}
```

Comment:
new line

Reply:
Fixed in 407f2376e1b33545f937f065467adcf94d65a6e2